content/en/tools/digital-currencies/defining/viewing-token-definitions.md

```diff
@@ -0,0 +1,47 @@
+---
+date: '2023-03-21'
+lastmod: '2023-03-21'
+section_menu: tools
+menu:
+  tools:
+    name: "Viewing Token Definitions"
+    weight: 400
+    parent: digital-currencies-token-definitions
+    identifier: digital-currencies-viewing-token-definitions
+    description: "Digital Currencies documentation describing how to create token definitions via the GUI"
+title: "Viewing Token Definitions"
+---
+
```

Comment:
```suggestion
## Viewing Token Definitions

```

Reply:
Don't think we need an initial heading as that automatically comes from the Title of the page. Let me know if this is otherwise.

Reply:
Don't think we need an initial heading as that automatically comes from the Title of the page. Let me know if this is otherwise.